ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.4.yaml

```diff
@@ -65,18 +65,8 @@ tests:
         SERVICE_BASE_REF=release-ocm-2.4
       TEST_SUITE: full
     workflow: baremetalds-assisted
-- as: e2e-metal-assisted-multi-version
-  optional: true
-  steps:
-    cluster_profile: packet-assisted
-    env:
-      ASSISTED_CONFIG: |
-        SERVICE_BASE_REF=release-ocm-2.4
-        OPENSHIFT_VERSION="all"
-    workflow: baremetalds-assisted
 - as: e2e-metal-assisted-olm
   optional: true
-  run_if_changed: .*operator.*
```

Comment:
you'll need to remove this configuration from the job as well

Reply:
It's not removed when running `make jobs`?

Reply:
no